### ISSUE TITLE
Check-in Data Errors

### ISSUE THREAD

**Issue by revsean** [samhowes/foothills-data#33]
Seems the check-in data that is used for the activity occurrence date is not always the date the check-in in PCO.  This is distorting the accurate Sunday count. 

 Example: Christine O'Donnell (27643344)'s checkin:224256728 has an occurred date of 10/4/2021 but the event she was checked in for was actually 9/13/2021. She was checked into that event on 10/4/2021. But we were backdating it. 
![Screen Shot 2021-10-05 at 7 52 37 AM](https://user-images.githubusercontent.com/86415336/136036654-cc0635cd-1e1c-4b02-9502-e97d6ae190cf.png)

Option: I can manually go through and redate these if that's easier. But I wonder if there is another date field that could be pulled from.

**Comment by revsean** [comment]
That makes sense. One of the ways this is showing up that I didn't understand is that in May we had something like 6.5K activities in Belongingandcommunity. Now, if you are using the created at data, that makes sense because that is when those check-ins were created by they were created over like 7 months. 

(Incidentally, I'd love to change the Channel for that specific event anyways from Belongingandcommunity to Caring, I'm guessing if we are going to run a new sych with the different date that can be accomplished then).

**Comment by samhowes** [comment]
Yep, I was using the "created_at" field. Instead, I can use the "starts_at" field of Event time.

Shown in the screenshot you posted are three "Event Times", "starting at" 9, 10, and 11. Using the "starts_at" field will make it look like people checked in at exactly the time the EventTime started. For live event data, that wouldn't be great, but for your case that seems like what you want to do. 

I've made the updates to the code and I'm ready to re-upload the data. The simplest thing for me to do is just delete all the checkin data and upload it fresh. So feel free to change all the channels and let me know when you'd like me to do the upload.

**Comment by samhowes** [comment]
I cleaned all the checkin data, however the report is still showing the checkin data. 
The report could just be cached and they haven't rebuilt it for whatever reason. But it is something for us to watch out for after I re-upload the data.

![image](https://user-images.githubusercontent.com/6216665/136698318-e697054e-93f8-434f-a2de-7c8690a6df99.png)

**Comment by samhowes** [comment]
Turns out there was more cleaning to do, I think I got it all this time.

@revsean Take a look at the new data for checkins and notes.
